src/ast/nodes/IfStatement.ts

```diff
@@ -82,6 +82,9 @@ export default class IfStatement extends StatementBase implements DeoptimizableE
 			}
 			if (this.alternate !== null) {
 				if (this.alternate.included) {
+					if (code.original.charCodeAt(this.alternate.start - 1) === 101 /* f */) {
```

Comment:
I'm always up for a good puzzle... What's the significance of `f`? Or is it `e`?
```
$ node -p 'String.fromCharCode(101)'
e
$ node -p 'String.fromCharCode(102)'
f
```

Reply:
The beauty of comments. I definitely did NOT copy this code from for..of loop 😉